examples/meta/src/evaluation/cross_validation.sg

```diff
@@ -0,0 +1,77 @@
+File f_feats_train = csv_file("../../data/classifier_binary_2d_linear_features_train.dat")
+File f_feats_test = csv_file("../../data/classifier_binary_2d_linear_features_test.dat")
+File f_labels_train = csv_file("../../data/classifier_binary_2d_linear_labels_train.dat")
+File f_labels_test = csv_file("../../data/classifier_binary_2d_linear_labels_test.dat")
+Math:init_random(2)
+
+#![create_features]
+Features features_train = features(f_feats_train)
+Labels labels_train = labels(f_labels_train)
+Features features_test = features(f_feats_test)
+Labels labels_test = labels(f_labels_test)
+#![create_features]
+
+#![set_parameters]
+real C = 1.0
+real epsilon = 0.001
+#![set_parameters]
+
+#![create_instance]
+Machine svm = machine("LibLinear", labels=labels_train, epsilon=epsilon, C1=C, liblinear_solver_type=enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC)
+#![create_instance]
+
+#![create_cross_validation]
+StratifiedCrossValidationSplitting splitting_strategy(labels_train, 2)
+Evaluation evaluation_criteron = evaluation("AccuracyMeasure")
+CrossValidation cross(svm, features_train, labels_train, splitting_strategy, evaluation_criteron, False)
+cross.set_num_runs(2)
+#![create_cross_validation]
+
+#![evaluate_and_get_result]
+CrossValidationResult result = cross.evaluate()
```

Comment:
here you need to make sure that you convert it with obtain_from_generic :(

Reply:
No need for the obtain_from_generic. Just dont use the specialized type
The method signature is `virtual CEvaluationResult* evaluate();`
so you can do
`EvaluationResult result = cross.evaluate()` and then you use `get` downstream.
With any of those changes **pls try them before you send a PR** as this back and forth is quite time consuming...